docs/otel/export-metrics.md

```diff
@@ -16,49 +16,152 @@ instruments not explicitly defined in the specification.
+  * [Metric: `otel.exporter.items`](#metric-otelexporteritems)
 
 <!-- tocstop -->
 
+## Principles used
+
+This specification defines three levels of detail possible, allowing
+for components to be used with `basic`, `normal`, and `detailed`
+levels that determine which attributes are kept or removed, as by a
+[Metric
+view](https://opentelemetry.io/docs/specs/otel/metrics/sdk/#view).  We
+rely on a conservation principle for pipelines, which states generally
+that what goes in, comes out.
+
+### Signal-independent metric names
+
+OpenTelemetry currently has 3 signal types, but it may add more.
+Instead of using the signal name in the metric names, we opt for a
+general-purpose noun that usefully describes any signal.  The
+signal-agnostic term used here is `items`, referring to spans, log
+records, and metric data points.  An attribute to distinguish the
+`signal` is used, with names designated by the project `traces`,
+`logs`, and `metrics`.
+
+Users are expected to understand that the data item for traces is a
+span, for logs is a record, and for metrics is a point.
+
+### Distinguishing Collectors from SDKs
+
+SDKs and Collectors process the same data in a pipeline, and both
+OpenTelemetry Collector and SDKs are recommended to use the metric
+names specified here.  An attribute to distinguish the `domain` is
+used, with values like `sdk`, `collector`.
+
+In a multi-level collection pipeline, each layer is expected to use a
+unique domain.  This enables calculating aggregates at each level in
+the collection pipeline and comparing them, as a measure of aggregate
+leakage.  Multi-level colletor topologies should allow configuration
+of distinct domains (e.g., `agent` and `gateway`).
+
+### Basic level of detail
+
+At the basic level of detail, we only need to know what goes in to a
+component, because we are able to infer a lot about the component by
+comparing its metrics with the next component in the pipeline.  By
+conservation, for example, any items that are received by a SDK
+processor component and are not received by the SDK exporter component
+must have been dropped.
+
+Therefore at the basic level of detail, all items of data are counted
+when they are done, regardless of the outcome.  No additional
+attributes are used at this level of detail.
+
+### Normal level of detail
+
+At the normal level of detail, an attribute is introduced that
+distinguishes whether the item was or was not successful.  A boolean
+attribute `success` is introduced at this level.
+
+It is understood that components have limited information about the
+success or failure of subsequent pipeline components.  While the
+veracity of `success=true` may be subject to reasonable doubt, the
+`success=false` attribute should be accepted as fact.  In a SDK
+configuration, the processor's `success=false` may be compared with
+the exporter's `success=false` to determine the number of items
+dropped by the processor, for example.
+
+### Detailed metrics
```

Comment:
kinda similar comment to basic - why have a separate level? What's the objective worth complicating the spec this way?

Reply:
This is about letting users trade costs based on what they need/want to observe: more metrics may be useful, but they're just additional expense when they're not being used.

I mentioned a personal side-story that led me to this realization in today's Spec SIG: to monitor a water system is similar to monitoring a telemetry pipeline, and it's also a situation where each individual metric is a substantial expense. The minimum, number of meters necessary to calculate total leakage in the system is 1 meter for (total) system production and 1 meter per user with a service connection. From total in and total out we can compute leakage, which is [equivalent with the calculation for dropped items ](https://github.com/carlosalberto/semantic-conventions/pull/1#discussion_r1370665614).

This leads to a conclusion that the minimum-cost configuration in for a telemetry pipeline, capable of computing a global Dropped statistic, would use Basic-level detail in each SDK, (disabled metrics for all intermediate collectors), and Normal-level detail for the final component of the final collector in the pipeline. If the user is in a situation where the metrics from the SDKs are not comparable with the metrics from subsequent stages in the pipeline for ay reason, they should use Normal-level detail in the SDK.

I'm also aware of tracing pipelines where there are rate limits enforced at the destination. This is a scenario where if the response code is `resource_exhausted` I should turn up sampling, if it's `timeout` I should complain to my backend team about an SLO violation, and if it's `queue_full` it means I should reconfigure the SDK.

Reply:
But why shouldn't this be handled with just the pre-aggregation rules ("views"?) instead of making it a problem for the exporters / components to know about different levels?

Reply:
(This content will appear in a new location, I'm writing an OTEP.)
My assumption is that this would be implemented using views, and the text of a semantic convention would be explaining which views to configure at which level of detail.

Reply:
Still rough: https://github.com/open-telemetry/oteps/pull/238